manpages/lumina-fileinfo.8

```diff
@@ -0,0 +1,27 @@
+.Dd October 24, 2017
+.Dt LUMINA-FILEINFO 8
+.Os 1.3.3
+
+.Sh NAME
+.Nm lumina-fileinfo
+.Nd is used to view the properties of specific files. It also includes a simple XDG application registration creator.
```

Comment:
Description feels a little long -> views properties of files. Includes creator for XDG application registrations.